### DISCUSSION TITLE
Error with registerSemanticSimilarityProvider

### DISCUSSION THREAD

**Discussion by pamelafox** [community/community#52832] [Copilot Conversations]
### Select Topic Area

Bug

### Body

This morning, I started seeing this error when Copilot tries to start up:

[ERROR] [default] [2023-04-14T15:00:42.621Z] GitHub Copilot could not connect to server. Extension activation failed: "Cannot read properties of undefined (reading 'registerSemanticSimilarityProvider')"

Notably, I'm in the Copilot X preview and using VS Code insiders with Copilot nightly. VS Code details:

Version: 1.78.0-insider (Universal)
Commit: 8f74fbfd1f2d8f6268a42df131726b218aafe511
Date: 2023-04-05T05:28:20.554Z
Electron: 22.4.3
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Darwin arm64 22.4.0
Sandboxed: Yes

**Comment by pamelafox** [comment]
It seems to have resolved once I restarted to update (twice).

**Comment by sarathavasarala-ms** [comment]
I'm facing this since this morning, still not working for me!
